configs/base_config.py

```diff
@@ -74,5 +74,9 @@ def RATELIMIT_STORAGE_URI(self):
     def SECRET_KEY(self):
         return self._get_config_value("SECRET_KEY")
 
+    @property
+    def DEFAULT_PAGE_SIZE(self):
```

Comment:
make sure to add it to SANDBOX Paramter store

Reply:
sandbox parameter store value have been added